builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -449,7 +450,12 @@ def unDeployService(stage) {
 			}
 			def envBucketKey = "${env_key}${configLoader.JAZZ.PLATFORM.AWS.S3.BUCKET_NAME_SUFFIX}"
 			def bucketObject = utilModule.getAccountBucketName(service_config);
-			sh "serverless remove --stage ${stage} --verbose  --bucket ${bucketObject[env_key]} --profile ${credsId}"
+			try {
+          sh "serverless remove --stage ${stage} --verbose  --bucket ${bucketObject[env_key]} --profile ${credsId}"
+      } catch (ex) {
```

Comment:
We are handling all exceptions this way. Should we handle only certain exceptions this way?

Reply:
Removed the generic error handling